plugin/mutate/source/dextool/plugin/mutate/backend/report/html/page_test_case_similarity.d

```diff
@@ -75,8 +95,11 @@ void toHtml(ref Database db, TestCaseSimilarityAnalyse result, Element root) {
                 " The difference column are the mutants that are only killed by the test case in the heading.");
 
     foreach (const tc; test_cases) {
-        root.addChild("h2", tc.name);
-        auto tbl = tmplDefaultTable(root, [
+        auto comp_container = root.addChild("div").addClass("comp_container");
+        comp_container.addChild("h2", tc.name).addClass("tbl_header");
+        auto tbl_container = comp_container.addChild("div").addClass("tbl_container");
+        tbl_container.setAttribute("style", "display: inline;");
```

Comment:
I think I understand what this is doing. It defaults the tables to being hidden, only show the header `h2`?
please add a oneline comment before line 98 that explain what you are doing (not the details, I can see the code) and why (<- the most important).